examples/slurm/prologMoneo.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/bash
 
-echo $SLURM_JOB_NODELIST > /tmp/jobid_$SLURM_JOB_ID.txt
+job_id="$SLURM_JOB_ID"
+job_name="$SLURM_JOB_NAME"
 
-# The command below pipes stdout/stderr to files, Moneo will still update the respective moneo logs 
-sudo -u <Your user> -H sh -c "cd  <Moneo Dir> &&  python3 moneo.py -j $SLURM_JOB_NAME.$SLURM_JOB_ID -c /tmp/jobid_$SLURM_JOB_ID.txt > /tmp/$SLURM_JOB_NAME.$SLURM_JOB_ID.Moneo.log 2>&1"
+if [ -z "$job_name" ]; then
+    job_name='slurm_job'
```

Comment:
Should we check for job id as well? Do NA if not set?

Reply:
The job_id is set by Slurm whereas the job_name is set by a user at submission time. It is possible that the user may not set the job name in there sbatch script so thats why I check. Job id should always be set by Slurm when the job is started. Would you like me to add the check anyways?

Reply:
Got it, make sense.